.gitignore

```
@@ -23,4 +23,6 @@ pubspec.lock
 .settings
 .last_build_id
 
-service-account.json
\ No newline at end of file
+service-account.json
+
+pubspec_overrides.yaml
```

```suggestion
**/pubspec_overrides.yaml
```
It should ignore all pubspec overrides files right?